File: JavaScript_Courses/JavaScript_Basics/JS_BP_2022_10/03_Conditional_Statements_Advanced_Exercise/01_Cinema.js
function cinema(input) {
    let premierePrice = 12.00;
    let normalPrice = 7.50;
    let discountPrice = 5.00;

    let ticketType = input[0];
    let rows = Number(input[1]);  // 10 Number
    let cols = Number(input[2]);  // 12 Number

    // let income = 0;
    let ticketPrice = 0;

    if (ticketType == "Premiere") {
        ticketPrice = premierePrice;
    } else if (ticketType == "Normal") {
        ticketPrice = normalPrice;
    } else if (ticketType == "Discount") {
        ticketPrice = discountPrice;
    }

    let income = rows * cols * ticketPrice;

    console.log(`${income.toFixed(2)} leva`);  // `
}

// cinema(["Premiere", "10", "12"]);

File: Python_Courses/Python_Basics/Python_PB_2025_01/03_Conditional_Statements_Advanced_Exercise/01_Cinema.py
# •	Premiere - премиерна прожекция, на цена 12.00 лева;
# •	Normal - стандартна прожекция, на цена 7.50 лева;
# •	Discount - прожекция за деца, ученици и студенти на намалена цена от 5.00 лева.

type_projection = input()
rows = int(input())
cols = int(input())

ticket_price = 0.0
if type_projection == 'Premiere':
    ticket_price = 12.00
elif type_projection == 'Normal':
    ticket_price = 7.50
elif type_projection == 'Discount':  # else:
    # else:
    ticket_price = 5.00

capacity = rows * cols  # 3r * 5c == 15

total_income = capacity * ticket_price

print(f'{total_income:.2f} leva')
